backend/clickhouse/metrics.go

```diff
@@ -2,58 +2,268 @@ package clickhouse
+	string(modelInputs.ReservedMetricKeyMetricName):        "MetricName",
+	string(modelInputs.ReservedMetricKeyType):              "Type",
+	string(modelInputs.ReservedMetricKeyTimestamp):         "Timestamp",
+	string(modelInputs.ReservedMetricKeyMetricDescription): "MetricDescription",
+	string(modelInputs.ReservedMetricKeyMetricUnit):        "MetricUnit",
+	string(modelInputs.ReservedMetricKeyStartTimestamp):    "StartTimestamp",
+	string(modelInputs.ReservedMetricKeyValue):             "Sum / Count",
+	string(modelInputs.ReservedMetricKeySecureSessionID):   "Exemplars.SecureSessionID",
+	string(modelInputs.ReservedMetricKeyTraceID):           "Exemplars.TraceID",
+	string(modelInputs.ReservedMetricKeySpanID):            "Exemplars.SpanID",
 }
 
-var metricsSamplingTableConfig = model.TableConfig{
-	AttributesColumns: metricsTableConfig.AttributesColumns,
-	BodyColumn:        metricsTableConfig.BodyColumn,
-	MetricColumn:      metricsTableConfig.MetricColumn,
-	KeysToColumns:     metricsTableConfig.KeysToColumns,
-	ReservedKeys:      metricsTableConfig.ReservedKeys,
-	SelectColumns:     metricsTableConfig.SelectColumns,
-	TableName:         TracesSamplingTable,
+var metricsArrayColumns = map[string]bool{
+	"Exemplars.SecureSessionID": true,
+	"Exemplars.TraceID":         true,
+	"Exemplars.SpanID":          true,
+}
+
+var metricsColumns = []string{
+	"ProjectId",
+	"ServiceName",
+	"MetricName",
+	"Type",
+	"Timestamp",
+	"MetricDescription",
+	"MetricUnit",
+	"Attributes",
+	"StartTimestamp",
+	"Exemplars.Attributes",
+	"Exemplars.Timestamp",
+	"Exemplars.Value",
+	"Exemplars.SecureSessionID",
+	"Exemplars.TraceID",
+	"Exemplars.SpanID",
+	"Min",
+	"Max",
+	"BucketCounts",
+	"ExplicitBounds",
+	"ValueAtQuantiles.Quantile",
+	"ValueAtQuantiles.Value",
+	"Count",
+	"Sum",
+}
+
+var MetricsTableNoDefaultConfig = model.TableConfig{
+	TableName:         MetricsTable,
+	KeysToColumns:     metricsKeysToColumns,
+	ArrayColumns:      metricsArrayColumns,
+	ReservedKeys:      reservedMetricsKeys,
+	BodyColumn:        "MetricName",
+	SelectColumns:     metricsColumns,
+	AttributesColumns: []model.ColumnMapping{{Column: "Attributes"}},
+}
+
+var MetricsTableConfig = model.TableConfig{
+	TableName:         MetricsTableNoDefaultConfig.TableName,
+	KeysToColumns:     MetricsTableNoDefaultConfig.KeysToColumns,
+	ArrayColumns:      MetricsTableNoDefaultConfig.ArrayColumns,
+	ReservedKeys:      MetricsTableNoDefaultConfig.ReservedKeys,
+	BodyColumn:        MetricsTableNoDefaultConfig.BodyColumn,
+	AttributesColumns: MetricsTableNoDefaultConfig.AttributesColumns,
+	SelectColumns:     MetricsTableNoDefaultConfig.SelectColumns,
+	DefaultFilter:     "",
 }
 
 var MetricsSampleableTableConfig = SampleableTableConfig{
-	tableConfig:         metricsTableConfig,
-	samplingTableConfig: metricsSamplingTableConfig,
-	sampleSizeRows:      20_000_000,
+	tableConfig:         MetricsTableConfig,
+	samplingTableConfig: MetricsTableConfig,
```

Comment:
I think we can delete `samplingTableConfig` and `sampleSizeRows` here if we're not doing any sampling